src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> To Do list </title>
+  <!-- <link rel="stylesheet" href="./style.css"> -->
+</head>
```

Comment:
- [ ] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)

Reply:
you are right thanks i will do that